models/iosxr/ospfv2/iosxr_ospfv2.yml

```diff
@@ -0,0 +1,1983 @@
+---
+GENERATOR_VERSION: "1.0"
+
+ANSIBLE_METADATA: |
+  {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'network'
+  }
+
+NETWORK_OS: iosxr
+RESOURCE: ospfv2
+COPYRIGHT: Copyright 2020 Red Hat
+
+DOCUMENTATION: |
+  module: iosxr_ospfv2
+  version_added: "2.10"
+  short_description: Manages OSPFv2 configuration on devices running Cisco IOS-XR.
+  description: 
+    - This module manages global OSPFv2 configuration on devices running Cisco IOS-XR.
+  author: Adharsh Srivats Rangarajan (@adharshsrivatsr)
+  options:
+    config:
+      description: A list of OSPF process configuration
+      type: list
+      elements: dict
+      suboptions:
+        processes: 
+          description: A list of OSPF instances configuration
+          type: list
+          elements: dict
+          suboptions:
+            address_family_unicast:
+              description: Enable unicast topology for ipv4 address family.
+              type: bool
+            adjacency_stagger:
+              description: Stagger OSPF adjacency bring up
+              type: dict
+              required: True
```

Comment:
Shouldn't the keys be required?

Reply:
Yeah, they are declared as required_together,
the command could be _adjacency stagger disable_ or _adjacency stagger < min va l> < max val >_